views/assets/src/tailwind.css

```
@@ -1,3 +1,6 @@
+/* Default UI font — Mona Sans (variable weight + italic), matching WP ERP. */
+@import url('https://fonts.googleapis.com/css2?family=Mona+Sans:ital,wght@0,200..900;1,200..900&display=swap');
```

Stylelint (17.13.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Mona+Sans:ital,wght@0,200..900;1,200..900&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Mona+Sans:ital,wght@0,200..900;1,200..900&display=swap'" (import-notation)

(import-notation)